src/renderer/renderScene.ts

```diff
@@ -252,6 +252,26 @@ export function renderScene(
     if (isOutOfBounds) {
       context.globalAlpha = 0.2;
     }
+
+    if (
+      sceneState.remotePointerActivity &&
+      sceneState.remotePointerActivity[clientId] === "down"
+    ) {
+      context.beginPath();
+      context.arc(x, y, 15, 0, 2 * Math.PI, false);
+      context.lineWidth = 3;
+      context.strokeStyle = "#ffffff88";
```

Comment:
does it support opacity?

Reply:
@lipis I think you can pass `rgba(..,..,..,0.5)` etc

Reply:
This one works like a charm..